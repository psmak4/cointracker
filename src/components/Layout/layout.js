import Container from 'react-bootstrap/Container'
import Header from './Header'

const Layout = ({ children }) => {
	return (
		<>
			<Header />
			<main>
				<Container fluid className='p-4'>
					{children}
				</Container>
			</main>
		</>
	)
}

export default Layout
