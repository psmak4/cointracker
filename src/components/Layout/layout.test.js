import { render, screen } from '@testing-library/react'
import Layout from './layout'

describe('<Layout />', () => {
	test('renders correctly', async () => {
		const sampleText = 'This is some sample text'
		const { container } = render(
			<Layout>
				<div>{sampleText}</div>
			</Layout>,
		)

		expect(container).toMatchSnapshot()

		const text = await screen.findByText(sampleText)
		expect(text).toBeInTheDocument()
	})
})
