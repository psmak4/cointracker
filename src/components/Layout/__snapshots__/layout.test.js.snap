// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders correctly 1`] = `
<div>
  <header
    class="navbar navbar-expand navbar-dark bg-dark"
    role="navigation"
  >
    <div
      class="px-4 container-fluid"
    >
      <span
        class="navbar-brand"
      >
        Cointracker
      </span>
    </div>
  </header>
  <main>
    <div
      class="p-4 container-fluid"
    >
      <div>
        This is some sample text
      </div>
    </div>
  </main>
</div>
`;
